### PR TITLE
To do List: Test

### PR DESCRIPTION
**Pull request by David-Verde**
Installed Jest, babel and Jsdom
Added tests for the add task function
Added tests for the delete task function
Mocks for local storage and DOM manipulation

### CODE REVIEW

File: src/modules/__tests__/add.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+describe('Add tests', () => {
+  test('Check task description', () => {
+    addTasks(taskArr);
+    expect(taskArr.length).toBe(1);
+  });
+
+  test('Check that a new task was created', () => {
+    expect(dom.window.document.querySelector('.txt')).toBeTruthy();
+  });
+});
```
Comment (FrederickMih) [inline src/modules/__tests__/add.test.js:105]: - [x] Hi @David-Verde, Some issues arises when I ran the test. Would you check out what's happening?. The screenshot below:

Screenshot from 2022-06-09 21-05-42

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️


- [ ] README.md
- Please let's use Grammarly when not sure of what were are typing. Your project title description is not well punctuated. 
- You haven't included instructions on how to set up the project locally in the **Getting Started** header of the template.
- The **MIT** license link of the **License** header is broken. 
- The link to your GitHub in the **Authors** header is equally broken.  

Please kindly address the issues above. Your README  file will be rendered more descriptive and professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/__tests__/add.test.js (hunk 1 above)
Comment (mikemtzp) [reply src/modules/__tests__/add.test.js:105]: Hi, @FrederickMih Thanks a lot for the review!😁
I deleted some previous configs in the `package.json` file. Now tests should work when you run `npm run test`

image

**Comment by David-Verde** [comment]
Edit README.MD
Add MIT license

**Review by KDlamini — APPROVED** [review]
### Hi team @David-Verde, @mikemtzp 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**